components/header/IconNavigation.tsx

```diff
@@ -0,0 +1,46 @@
+import type { AvailableIcons } from "$store/components/ui/Icon.tsx";
+import Icon from "$store/components/ui/Icon.tsx";
+
+export interface IconNavigation {
+  title: string;
+  href: string;
+  icon: AvailableIcons;
+  mobileVisibility?: boolean;
+}
+
+interface Props {
+  items?: IconNavigation[];
+}
+
+export const IconNavigation = ({ items }: Props) => {
+  if (!items?.length) {
+    return null;
+  }
+
+  return (
+    <div class="flex flex-row  gap-x-4  justify-center items-baseline  lg:items-center ">
```

Comment:
As vezes voce coloca dois `  ` espaços nas classes, cuida com isso
```suggestion
    <div class="flex flex-row gap-x-4 justify-center items-baseline lg:items-center">
```